apps/server/src/user/user.service.ts

```diff
@@ -21,6 +21,12 @@ import {
   UserSignupDtoType,
   UserUpdateDtoType,
 } from './dto/user.dto';
+import { User } from '@prisma/client';
+
+export interface LoginResponse {
```

Comment:
@uptownhr lets keep it in the same User namespace:
UserLoginResponse.

Reply:
updated